homeassistant/components/weather/__init__.py

```diff
@@ -35,6 +35,7 @@
 ATTR_FORECAST_CONDITION = "condition"
 ATTR_FORECAST_PRECIPITATION = "precipitation"
 ATTR_FORECAST_PRECIPITATION_PROBABILITY = "precipitation_probability"
+ATTR_FORECAST_PRESSURE = "pressure"
```

Comment:
Changing the entity model requires first approval in an issue in our architecture repo.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

Reply:
Ah OK. I have created an issue [here](https://github.com/home-assistant/architecture/issues/467).

Reply:
@MartinHjelmare  I'm new to this project - what is the next step?

Reply:
We need to get acceptance in the architecture issue first. Then when that is done, we need to open PRs in relevant repos, this one, the frontend, the developer docs and possibly the user docs.

Reply:
The architecture issue is approved. Please open the other needed PRs as mentioned above.

Reply:
Would it just be one additional PR - moving the architecture change to its own PR (homeassistant/components/weather/__init__.py)?

Reply:
- We need one PR for the core with the addition here above.
- We need to update our dev docs for the weather entity: https://developers.home-assistant.io/docs/core/entity/weather#forecast
- We may need to update the frontend weather card. Please check the frontend source and ask the frontend team about this.

Reply:
OK here's the core addition: https://github.com/home-assistant/core/pull/44965

Reply:
Documentation: https://github.com/home-assistant/developers.home-assistant/pull/768

Reply:
RE:frontend, I asked in #devs_frontend and they say it shouldn't break anything there. If someone wants to add it to a card, that should be doable, but I don't think this needs to wait on that.